### ISSUE TITLE
Crash using Solar Panel to power adjacent Alloy Furnace (1.11.2)

### ISSUE THREAD

**Issue by Auropen** [turbodiesel4598/NuclearCraft#114]
Tried this the latest version of this mod, in a small 1.11.2 modpack, and crashed.

Crash happend when using a SolarPanel next to an Alloy Furnace, as soon as I changed the time to get daylight.

Suppose the CoFH RF API is still missing in 1.11.2?

```
java.lang.NoSuchMethodError: cofh.api.energy.IEnergyReceiver.receiveEnergy(Lnet/minecraft/util/EnumFacing;IZ)I
	at nc.tile.energy.TileEnergy.pushEnergy(TileEnergy.java:168)
	at nc.tile.generator.TilePassiveGenerator.func_73660_a(TilePassiveGenerator.java:24)
	at net.minecraft.world.World.func_72939_s(World.java:1802)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:621)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:720)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:624)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:149)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:482)
	at java.lang.Thread.run(Unknown Source)
```

Edit:
Just saw that CoFH Energy API is in the source codes, might be a fixable bug then?

**Comment by Auropen** [comment]
Trying to reproduce the crash, seems to be working now.. both using cable from Mekanism and just placing adjacent NeclearCraft machines, seem to transfer the RF correctly from a Basic Solar Panel.

**Comment by turbodiesel4598** [comment]
This should now be fixed in 2.3 :)
